draft-ietf-tls-tls13.md

```diff
@@ -2244,16 +2244,18 @@ SignatureSchemeList value:
            ed25519(0x0807),
            ed448(0x0808),
 
+           /* Legacy algorithms */
+           ecdsa_sha1(0x0203),
+
            /* Reserved Code Points */
-           dsa_sha1_RESERVED(0x0202),
-           dsa_sha256_RESERVED(0x0402),
-           dsa_sha384_RESERVED(0x0502),
-           dsa_sha512_RESERVED(0x0602),
-           ecdsa_sha1_RESERVED(0x0203),
            obsolete_RESERVED(0x0000..0x0200),
+           obsolete_RESERVED(0x0202),
```

Comment:
I don't really want to strip the labels off of these; they have prior meaning, and there's no reason to obfuscate it here. (they're already hidden up top and only in the appendix) What's the reason for this proposed change?